docs/source/usecases/VirtualFunctions.md

```diff
@@ -111,6 +111,61 @@ deviceInstance->setAField(someHostValue); // set some field on the host
 
 This is the solution that the code teams we have talked to have said is the most productive way to solve the problem.
 
+## But what if I do not really need the V-Tables on the device side?
+
+The performance critical part of a code might not use pointers to base class and dynamic polymorphism to compute a result from data.
+Thus, the device might not need to have a working virtual function table.
+Consider the following example:
+```c++
+struct Interface
+{
+    KOKKOS_DEFAULTED_FUNCTION
+    virtual ~Interface() = default;
+
+    KOKKOS_FUNCTION
+    virtual void operator()( const size_t) const = 0;
+};
+
+struct Implementation : public Interface
+{
+    KOKKOS_FUNCTION
+    void operator()(const size_t i) const override
+    { ... }
+
+    void apply(){
+        Kokkos::parallel_for("myLoop",10,
+            KOKKOS_CLASS_LAMBDA (const size_t i) { this->operator()(i); }
+        );}
+};
+
+int main ()
+{
+    ... 
+    auto implementationPtr_h = std::make_shared<Implementation>();
+    implementationPtr_h->apply();
+}
+```
+### What is the problem?
+
+Inside the `parallel_for` the `operator()` is called which is marked `override`.
```

Comment:
Sentence reads weird

Reply:
is [900707](https://github.com/kokkos/kokkos-core-wiki/pull/162/commits/9007071808750d512375291dbaedc433bbb1b9e8) better?